src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+promise1
+  .then(() => {
+    document.body.insertAdjacentHTML('beforeend', `
+    <div class="message">Promise was resolved!</div>
+    `);
```

Comment:
you have written the same code several times. please create separate function `createNotification` with params and use it everywhere

Reply:
i think, that i right understood